src/textual/dom.py

```diff
@@ -83,6 +83,26 @@ class NoScreen(DOMError):
     pass
 
 
+class _ClassesDescriptor:
```

Comment:
Mypy doesn't like it if you have a property where the setter and getter have different types. However, it is happy with a descriptor.